default_organisation.bash

```diff
@@ -86,16 +86,34 @@ set_gpg_recipients() {
 		while read gpg_id; do
 			# Check that gpg know recipient and policy allow to encrypt for them
 			local can_encrypt=0
+			$GPG --list-key "$gpg_id" > /dev/null
+			if ! [[ $? -eq 0 ]]; then
+				echo ""
+				echo "Fingerprint \"$gpg_id\" was not found. Maybe it is not imported."
+				read -r -p "Do you want to import it? [y/N] " input < /dev/tty
+				case $input in
+					[y/Y]) gpg --recv-keys "$gpg_id" || exit 1 ;;
+				esac
```

Comment:
To keep consistency with `pass`, I would prefer:

```bash
read -r -p "Do you want to import it? [y/N] " response
[[ $response == [yY] ]] && $GPG --recv-keys "$gpg_id" || exit 1
````

Reply:
Do you need `< /dev/tty` ? This can maybe be an issue on certain platform?

Reply:
Without the `< /dev/tty`, `read` is reading the response from the output of GPG, then the prompt does not appear.